vars/skipStage.groovy

```diff
@@ -523,6 +523,12 @@ boolean call(Map config = [:]) {
         case 'Functional_Hardware_Medium':
         case 'Functional Hardware Medium':
             return skip_ftest_hw('medium', target_branch)
+        case 'Functional_Hardware_Medium_Verbs_Provider':
```

Comment:
Not necessary.  Was only there for other stages for hysterical raisins.

Reply:
Removed